independent-publisher-connectors/Google Books/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1243 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Google Books",
+        "description": "Search and discover data and details about and within books available in the vast library of Google",
+        "version": "1.0",
+        "contact": {
+            "name": "Fördős András",
+            "email": "fordosa90@gmail.com"
```

Comment:
@fordosa90 please provide a "url" property in the contact section.
"url": "Contact URL",